web/src/engine/osk/src/visualKeyboard.ts

```diff
@@ -804,9 +804,12 @@ export default class VisualKeyboard extends EventEmitter<EventMap> implements Ke
   }
 
   get layoutHeight(): ParsedLengthStyle {
+    // currently:  equivalent to if(is touch keyboard)
     if (this.usesFixedHeightScaling) {
-      let baseHeight = this.height;
+      let baseHeight = this.height; // initially undefined!
       baseHeight -= this._borderWidth * 2;
+      // For robustness, as there are legal cases where this.height may be undefined.
+      baseHeight = (isNaN(baseHeight) || baseHeight < 0) ? 0 : baseHeight;
```

Comment:
To be clear, it shouldn't be `undefined` when this is called, but without the other changes in this PR, it could be - which is what caused the motivating Sentry report.